saleor/graphql/schema.graphql

```diff
@@ -27426,6 +27429,40 @@ enum VolumeUnitsEnum {
   ACRE_FT
 }
 
+"""
+Event sent when account confirmation requested.
+
+Added in Saleor 3.15.
+"""
+type AccountConfirmationRequested implements Event @doc(category: "Users") {
```

Comment:
It does not include information if the confirmation is required to log in or optional.